AWGL/somatic_db
# Myeloid gaps summary changes

## Change description
- Add a summary of the regions that aren't covered to either 0X or 270X, as required by the haem onc team for moving myeloid onto TSO500
- Summary will only be shown for myeloid panels (CLL, MPN and Myeloid)
- The models have been changed slightly, there is an extra field in the Panel model to set whether a panel is haem onc or not

## Justification for minor update
No effect on processing of data, only how it's visualised

## Testing description
- Load in fixtures and run unit tests
- Test migrations for the new model change, should work without any issues as there is a default set
- Check that a myeloid referral shows the correct gaps summary in the Depth tab, the Report Summary tab and the PDF report download
- Check that a non-myeloid referral displays the usual gaps in the above tabs

## Test data location
N/A

## Verification results
All tests passed without issue
